### ISSUE TITLE
Component Build: Grid Item

### ISSUE THREAD

**Issue by BostonRohan** [Juxtify4Spotify/juxtify-client#18]
Grid item should be able to be extended to any category. (Playlists, Folders, Artists, Albums)

https://www.figma.com/design/77UhOWnu7BZjrBMWmQZGxf/Spotify-Redesign-(Community)-(Copy)?node-id=217-41052&t=DrzBl1przRKnnVVV-4

**Comment by RichardKanshen** [comment]
sooooo this one, right?
![image](https://github.com/Juxtify4Spotify/juxtify-client/assets/73738591/0225a353-1250-471d-9afc-0323f8db1b49)

**Comment by BostonRohan** [comment]
@RichardKanshen Yeah that's what I'm thinking. And then passing down those elements as props so we can use them for the other categories. What do you think?

**Comment by RichardKanshen** [comment]
Sounds alright!
